packages/taquito/src/context.ts

```diff
@@ -206,24 +219,9 @@ export class Context {
     // We reduce the default value in the same proportion, from 10 to 5.
     const defaultInterval = 5;
     try {
-      const constants = await this.rpc.getConstants();
-      let blockTime = constants.time_between_blocks[0];
-      if (constants.minimal_block_delay !== undefined) {
-        blockTime = constants.minimal_block_delay;
-      }
-      let confirmationPollingInterval = BigNumber.sum(
-        blockTime,
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        new BigNumber(constants.delay_per_missing_endorsement!).multipliedBy(
-          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-          Math.max(0, constants.initial_endorsers! - constants.endorsers_per_block)
-        )
-      );
-
-      // Divide the polling interval by a constant 3
-      // to improvise for polling time to work in prod,
-      // testnet and sandbox enviornment.
-      confirmationPollingInterval = confirmationPollingInterval.dividedBy(3);
+      const constants = await this.readProvider.getProtocolConstants('head');
+      const blockTime = constants.minimal_block_delay ? constants.minimal_block_delay : constants.time_between_blocks ? constants.time_between_blocks[0] : new BigNumber(defaultInterval);
```

Comment:
can I get a brief explanation on what blockTime is used for?

Reply:
It is used to calculate the polling interval used during the operation confirmation